db/queries/assessment_records/queries.py

```diff
@@ -95,7 +96,12 @@ def get_metadata_for_fund_round_id(
     statement = (
         select(AssessmentRecord)
         # Dont load json into memory
-        .options(defer(AssessmentRecord.jsonb_blob)).where(
+        .options(
+            defer(AssessmentRecord.jsonb_blob),
+            selectinload(AssessmentRecord.qa_complete),
```

Comment:
I think they're "lazy loaded" by default because I noticed when calling `AssessmentRecord.qa_complete` (or .flags or .tag_associations) it was emitting a select call. And doing that for each object ended up with an N+1 query problem. By using selectinload it definitely reduced the number of queries and overall db time.

Reply:
ah interesting, its because its a relationship: https://github.com/communitiesuk/funding-service-design-assessment-store/blob/1ffe0258311f0f50985b9330424d6e845324f589/db/models/assessment_record/assessment_records.py#L69-L70
In SQLAlchemy, relationships are lazy-loaded by default 👍